.github/workflows/build.yml

```diff
@@ -114,3 +114,15 @@ jobs:
               make agent-rest-docker
               docker tag aries-framework-go/agent-rest:latest  docker.pkg.github.com/hyperledger/aries-framework-go/agent-rest:$AGENT_IMAGE_TAG
               docker push docker.pkg.github.com/hyperledger/aries-framework-go/agent-rest:$AGENT_IMAGE_TAG
+
+      - name: Publish npm packages
```

Comment:
@fqutishat @llorllale Don't we need the logic to differentiate between snapshots and releases?

Reply:
yes

Reply:
https://github.com/hyperledger/aries-framework-go/pull/1197#discussion_r376084176